### PR TITLE
jasper: security patches

### PR DESCRIPTION
**Pull request by nijikon**

### CODE REVIEW

File: Library/Formula/jasper.rb [hunk 1]
```diff
@@ -23,11 +24,56 @@ class Jasper < Formula
     cause "Undefined symbols when linking"
   end
 
+  # CVE-2011-4516: heap-based buffer overflow
+  # CVE-2011-4517: heap-based buffer overflow
+  # CVE-2014-8137: double-free
+  # CVE-2014-8138: heap-based buffer overflow
+  # CVE-2014-8157: off-by-one
+  # CVE-2014-8158: multiple stack-based buffer overflows
+  # CVE-2014-9029: multiple off-by-one
```
Comment (mikemcquaid) [inline Library/Formula/jasper.rb]: What's the source for these?

Comment (mikemcquaid) [inline Library/Formula/jasper.rb]: Would be good to submit to https://github.com/homebrew/patches instead of Gists too.

Comment (nijikon) [inline Library/Formula/jasper.rb]: http://cvsweb.openbsd.org/cgi-bin/cvsweb/ports/graphics/jasper/patches/

Comment (nijikon) [inline Library/Formula/jasper.rb]: Can submit them to patches, did not know that it exists :)

Comment (mikemcquaid) [inline Library/Formula/jasper.rb]: Have they been submitted upstream?

Comment (DomT4) [inline Library/Formula/jasper.rb]: Ping?

**Comment by DomT4** [comment]
Ping again? It'd be nice to get this done.

**Comment by apjanke** [comment]
Closing due to legacy-homebrew being deprecated. Please re-open on the new homebrew/core repo if upstream takes the patches.

**Pull request closed without merging**